crates/next-core/Cargo.toml

```diff
@@ -11,7 +11,9 @@ bench = false
 [dependencies]
 anyhow = "1.0.47"
 indexmap = { workspace = true, features = ["serde"] }
+lazy_static = "1.4.0"
```

Comment:
Nit: I think we're standardizing on `once_cell`, because it matches the standardized value: https://github.com/rust-lang/rust/issues/74465

Reply:
Looks like we use the `once_cell` crate mostly, but we use the unstable compiler feature in some of the examples files (interestingly, these are the older of the two). I'll use the crate here.